pubtools/_quay/signer_wrapper.py

```diff
@@ -358,6 +355,97 @@ def remove_signatures(
         self._remove_signatures(to_remove)
 
 
+class CosignSignerSettingsSchema(Schema):
+    """Validation schema for cosign signer settings."""
+
+    quay_namespace = fields.String(required=True)
+    quay_host = fields.String(required=True)
+    dest_quay_api_token = fields.String(required=True)
+
+
+class CosignSignerWrapper(SignerWrapper):
```

Comment:
This wrapper doesn't create signatures, only removes them?

Reply:
Signatures are created by sign_container method in the parent class